### DISCUSSION TITLE
Error using cohere embeddings in qachain

### DISCUSSION THREAD

**Discussion by mashedp0tato** [FlowiseAI/Flowise#161] [Q&A]
I was trying to make a simple Conversational Retrieval QA Chain where gpt3.5 turbo is used as llm, pinecone as vectorstore and cohere's multilingual v2 as embeddings model. I could successfully generate and store the embeddings in Pinecone. But then got
'this.retriever.getRelevantDocuments is not a function' as error 

Any idea how to make flowise work with Cohere embeddings?

**Comment by HenryHengZJ** [comment]
hey it works fine for me when using Cohere embeddins. Below is the chatflow I was using, try copy the below content, and paste it onto a new canvas

[CohereEmbeddings.txt](https://github.com/FlowiseAI/Flowise/files/11555002/CohereEmbeddings.txt)
